tests/conftest.py

```diff
@@ -21,24 +21,32 @@ def _to_module_string(path: str) -> str:
 ]
 
 
-def pytest_configure():
-    """Perform initial configuration for pytest.
-    Fetches pose data file paths as a dictionary for tests.
-    """
+def pytest_sessionstart(session):
+    """Set up logging to file and fetch pose data file paths."""
+    # Set up log file in a temporary directory
+    tmp_path_factory = session.config._tmp_path_factory
+    pytest.LOG_FILE = logger.configure(
+        log_file_name=".movement-test",
+        log_directory=tmp_path_factory.mktemp(".movement"),
+        console=False,
+    )
+    # Fetch pose data file paths as a dictionary
     pytest.DATA_PATHS = {}
     for file_name in list_datasets():
         paths_dict = fetch_dataset_paths(file_name)
         data_path = paths_dict.get("poses") or paths_dict.get("bboxes")
         pytest.DATA_PATHS[file_name] = data_path
 
 
-@pytest.fixture(autouse=True)
-def setup_logging(tmp_path):
-    """Set up logging for the test module.
-    Redirects all logging to a temporary directory.
-    """
-    configure_logging(
-        log_level=logging.DEBUG,
-        logger_name="movement",
-        log_directory=(tmp_path / ".movement"),
+@pytest.fixture
+def caplog(caplog: LogCaptureFixture):
```

Comment:
I'm not sure I fully understand what this override does, but it seems to allow us stick to all of our existing uses of `caplog` in the tests, so I'm happy.

Reply:
Added more details to clarify
https://github.com/neuroinformatics-unit/movement/blob/deb55013d2a4162970f0b5cd6ea55e3875f59308/tests/conftest.py#L42-L45